lib/utils/date_time_utils.dart

```diff
@@ -0,0 +1,3 @@
+class DateTimeUtils {
+  static int get unixTimeNow => DateTime.now().millisecondsSinceEpoch ~/ 1000;
```

Comment:
Should we add a method to identify large discrepancies between local time and node time?